src/uct/ib/mlx5/ib_mlx5_log.c

```diff
@@ -61,6 +61,10 @@ ucs_status_t uct_ib_mlx5_completion_with_err(uct_ib_iface_t *iface,
     uint32_t     qp_num;
     void         *wqe;
 
+    if (ecqe == NULL) {
+        return UCS_OK;
```

Comment:
can you explain this line please?

Reply:
Will comment the code, thanks

Reply:
need to avoid calling log function from uct_dc_mlx5_ep_handle_failure
uct_rc_mlx5_iface_commom_clean, called from uct_dc_mlx5_iface_reset_dci, should remove all err completions during reset